lib/versions.js

```diff
@@ -60,3 +60,15 @@ function fetchLatestTag(repo) {
     },
   );
 }
+
+function getVersionFromToolVersionsFile() {
+  const fileName = ".tool-versions";
+  if (!fs.existsSync(fileName)) return;
+
+  const toolVersions = fs.readFileSync(fileName, "utf-8").split("\n");
+  const scarbVersionInfo = toolVersions.find((tool) => tool.match(/^scarb/));
```

Comment:
nitpick, [`match` returns `null` if no matches are found](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/match#return_value), so this `find` call can be removed

I'm talking about this not for micro-optimization purposes; but rather I'm afraid of having two different regexes being used here